projects/core/src/occ/occ-models/occ.models.ts

```diff
@@ -2219,6 +2219,27 @@ export interface OrderStatusUpdateElementList {
   orderStatusUpdateElements?: OrderStatusUpdateElement[];
 }
 
+export interface ConsignmentTrackingEvent {
```

Comment:
This `occ.models.ts` is generated automatically from the backend/swagger file, so it might be better to move those interfaces to separate file, either to this folder, or to `user\model` one